### ISSUE TITLE
Why not use bidirectional LSTM?

### ISSUE THREAD

**Issue by liuhu-bigeye** [yalecyu/crnn.caffe#2]
I found that in the [origin CRNN implementation](https://github.com/bgshih/crnn/blob/master/model/crnn_demo/config.lua), the author used 2 layers of BLSTM, but i only found 2 layers of LSTM. Would you please provide some help for that implementation?

**Comment by yalecyu** [comment]
Maybe,I will consider this content later. Summer vacation is coming!

**Comment by stalagmite7** [comment]
@liuhu-bigeye , @yalecyu , have you been successful in writing a bidirectional LSTM for the net? Is it possible to just add a reverse layer for the LSTM? I am new to caffe so don't really know how to implement my own layers, but if you could point me to some resources I could use to do that, I'd be very grateful!
Thanks!

**Comment by yalecyu** [comment]
@stalagmite7 No,LSTM is work well too. BLSTM is not an urgent thing.If you are interested in the project,you can try.
